modules/cloud-provider/proxmox.nix

```diff
@@ -0,0 +1,31 @@
+# Proxmox PVE settings
+# Processors: CPU type host
+# BIOS: OVMF (UEFI)
+# Machine: Default (i440fx)
+# SCSI Controller: VirtIO SCSI
+# Hard Disk: VirtIO Block
+
```

Comment:
Does this still apply for us?

Reply:
Yes!
Those settings need to be set on the VM host.
The module works under the assumption everything is configured that way.